packages/iris-grid/src/IrisGridTableModel.ts

```diff
@@ -296,6 +300,10 @@ class IrisGridTableModel
     );
   }
 
+  get isPartitionAwareSourceTable(): boolean {
+    return true;
```

Comment:
To be safe, should check `isPartitionRequired` as well:
```suggestion
    return this.isPartitionRequired && true;
```